.github/workflows/release-to-marketplaces.yml

```diff
@@ -0,0 +1,36 @@
+name: Release Extension to official Marketplace and Open VSX registry
+on: 
+    # It cannot run on release event as when release is created then version is not yet bumped in package.json
+    push:
+      branches:
+        - master
+
+jobs:
+
+  release:
+    name: Release to marketplaces
+    # such a specific commit messages comes from if-nodejs-version-bump.yml workflow
+    if: startsWith(github.event.commits[0].message, 'chore(release):')
```

Comment:
this workflow runs on push because it cannot rely on release event.

so we create a GitHub package with release workflow that is already in the repo (now added dependencies to enable it) and then bump version in package.json with another workflow that creates a commit `chore(release):`

I hope that makes sense, sounds complicated but thanks to it we have one universal pipeline for all the repos